<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>RUN</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultsTextEdit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>581</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="algorithmComboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>DRH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Nieuw item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delta(+RLE)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LZW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Huffman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delta+LZW</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Delta+Huffman</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sprintz</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LZ77</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>LZ4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>zstd</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lzma</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>lzo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>snappy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>(SW-)RLE</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dataset</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fileLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>C:/Users/arthu/CLionProjects/Thesis/Simplified_dataset1.txt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveCsvButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Save to CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizeButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Visualize</string>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="adaptiveCheckBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adaptive Window</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="windowSizeSpinBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fixed Window size:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="visualizeGraphButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>390</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Graphs</string>
    </property>
   </widget>
   <widget class="QFrame" name="framePlots">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>541</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
